src/taker/api.rs

```diff
@@ -2061,39 +2103,85 @@ impl Taker {
                 }
             };
 
+        // Define the cache file name. (Using a binary file for bincode.)
+        let cache_file = get_taker_dir().join("offerbook_cache.json");
```

Comment:
This file is differnt from where the offerdata is being stored, in `offerbook.dat`. We can just use one file for everything. Call that `offerbook.json`.